kubernetes/razeedash/resource.yaml

```diff
@@ -30,6 +30,9 @@ items:
             app: razeedash
           name: razeedash
         spec:
+          securityContext:
+            fsGroup: 999
```

Comment:
Just curious.  We've been using 1000 for the GIDs and UIDs.  Any reason you use 999 or is it just habit?

Reply:
Just a habit, maybe a bad one. Sorry did not see this message until now